src/applications/mhv/secure-messaging/tests/components/Compose/ComposeForm.unit.spec.jsx

```diff
@@ -687,6 +691,28 @@ describe('Compose form component', () => {
       'trigger',
       "You're not connected to any care teams in this messaging tool",
     );
+    const viewOnlyDraftSections = screen.queryAllByTestId(
+      'view-only-draft-section',
+    );
+    expect(viewOnlyDraftSections.length).to.equal(3);
+    viewOnlyDraftSections.forEach((draftSection, index) => {
+      expect(draftSection.firstChild.tagName).to.equal('STRONG');
+      if (index === 0) {
+        expect(draftSection.lastChild.textContent).to.equal(
+          'COVID: Ask COVID related questions',
+        );
+      }
+      if (index === 1) {
+        expect(draftSection.lastChild.textContent).to.equal(
+          customState.sm.threadDetails.drafts[0].subject,
+        );
+      }
+      if (index === 2) {
+        expect(draftSection.lastChild.textContent).to.equal(
+          customState.sm.threadDetails.drafts[0].messageBody,
+        );
+      }
+    });
```

Comment:
can you please elaborate on this statement, I am not sure I am following the use of index here

Reply:
I created one component that has a test-id of "view-only-draft-section" and I'm using that component to render the category, subject, and message body, so when I query that test-id, I get an array of 3.  I wanted to compare the rendered text to the text stored in the drafts[0] object, and each of the 3 elements is at a diferent index in the array.

Reply:
aha, okay, that makes sense now. Thanks